web/Makefile

```diff
@@ -48,6 +48,7 @@ scripts: builddir npminit
 	    | ./try-uglify ${BUILDDIR}/scripts/jquery-slickgrid-2.2.6.min.js
 
 css: builddir
+	sass css/style.scss css/style.css --style compressed
```

Comment:
I don't have the command `sass` on my system. I could install it from my distribution's package manager, but there is a version of sass in npm as `node-sass`. Could we add that to the `web/package.json` `devDependencies` list, and use that version instead? Then nobody has to worry about whether or not they installed a local copy.